cdc/generation.cc

```diff
@@ -292,6 +295,21 @@ future<utils::chunked_vector<mutation>> get_cdc_generation_mutations(
         co_await coroutine::maybe_yield();
     }
 
+    result_size_estimate += size_estimate;
+
+    utils::get_local_injector().inject("cdc_generation_mutations_replication", [&res, result_size_estimate, mutation_size_threshold] {
+        utils::chunked_vector<mutation> new_res;
+
+        size_t number_of_copies = (mutation_size_threshold / result_size_estimate + 1) * 2;
```

Comment:
Please write a comment explaining what this calculation aims to achieve. (The commit message provides a nice explanation; but the commit message is lost in code)

Reply:
Done.